docs/examples/personal-travel-assistant.mdx

```diff
@@ -19,7 +19,99 @@ pip install openai mem0ai
 
 Here's the complete code to create and interact with a Personalized AI Travel Assistant using Mem0:
 
-```python
+<CodeGroup>
+
+```after v1.1
```

Comment:
can you capitalize the first letter of before and after?

Reply:
Yes, also added the code type for highlights.